src/components/PlantisScreens.jsx

```diff
@@ -1,8 +1,175 @@
 import React from 'react';
-import plantisHomepage from '../plantisHomepage.png';
+import YouTube from 'react-youtube';
```

Comment:
This file is a little big, you may want to think about breaking it up in future

Reply:
thanks, yes, that was my next step - to split into components - I just wanted to get the page up and running